CasperSleep/nightshade
# 122-part-2-use-yaml-for-pattern-pages

Closes #122
- Move completed pattern page data objects to yaml
- Clean-up and simplify data structure
- Changes to macros based on data structure changes
  - Simplify how data is passed and handled
  - Add checks so patterns look good if data is incomplete
### Todo
- [ ] Remove squashed commit once #142 is merged
- [ ] Document macros
### Test
1. `rm -rf dist` folder to start from clean slate
2. Run `gulp`
3. Navigate to popovers, tooltips, buttons, banners. Content should match what's live on `nightshade.rocks`.
